docs/system-info/custom-portals-services-and-rates.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
fix: please remove this page - this is duplicative of the rates published in the [Box link](https://wustl.box.com/s/umzho1jlyoxj2agbuee77ic8xkk2z4jw), and this would make it public to everyone - even outside the university.